### PR TITLE
Endret response til json for initialize og upload

### PR DESCRIPTION
**Pull request by mSunberg**
## Description
This code changes the response for initialize and upload from text to json.

## Related Issue(s)
- #{https://github.com/Altinn/altinn-broker/issues/542}

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

### CODE REVIEW

**Comment by Ceredron** [comment]
Looks like a lot of tests are failing. I think it is because FileTransferId has changed from string to object so everywhere that gets a FileTransferId needs to update. But it should only be a few helper functions. For instance, in FileTransferControllerTests/InitializeAndAssertBasicFileTransfer
It tries to get FileTransferId like this:
        return await initializeFileTransferResponse.Content.ReadAsStringAsync();
But should be:
        var fileTransferResponse = await initializeFileTransferResponse.Content.ReadFromJsonAsync<InitializeFileTransferResponseExt>();
        return fileTransferResponse.FileTransferId;

**Review by Ceredron — COMMENTED** [review]
Looks like a lot of tests are failing. I think it is because FileTransferId has changed from string to object so everywhere that gets a FileTransferId needs to update. But it should only be a few helper functions. For instance, in FileTransferControllerTests/InitializeAndAssertBasicFileTransfer
It tries to get FileTransferId like this:
return await initializeFileTransferResponse.Content.ReadAsStringAsync();
But should be:
var fileTransferResponse = await initializeFileTransferResponse.Content.ReadFromJsonAsync();
return fileTransferResponse.FileTransferId;

File: src/Altinn.Broker.API/Controllers/FileTransferController.cs [hunk 1]
```diff
@@ -78,7 +82,10 @@ CancellationToken cancellationToken
             ContentLength = Request.ContentLength ?? Request.Body.Length
         }, cancellationToken);
         return commandResult.Match(
-            fileId => Ok(fileId.ToString()),
+            fileId => Ok(new FileTransferUploadResponseExt()
+            {
+                FileId = fileId
+            }),
```
Comment (RagnarFatland-Avanade) [inline src/Altinn.Broker.API/Controllers/FileTransferController.cs:88]: Consider correcting this naming.
"fileId" is an outdated term, the correct name is "fileTransferId"
(In this case the property name was not previousy exposed in the API, but with this change it becomes; so therefore naming becomes importan).

File: src/Altinn.Broker.API/Models/FileTransferUploadResponseExt.cs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+namespace Altinn.Broker.API.Models;
+
+public class FileTransferUploadResponseExt
+{
+    public Guid FileId { get; set; }
```
Comment (RagnarFatland-Avanade) [inline src/Altinn.Broker.API/Models/FileTransferUploadResponseExt.cs:5]: Consider correcting this naming.
"fileId" is an outdated term, the correct name is "fileTransferId"
(In this case the property name was not previousy exposed in the API, but with this change it becomes; so therefore naming becomes importan).

**Review by Ceredron — APPROVED** [review]

**Pull request merged**